AEPIdentity/Sources/IdentityState.swift

```diff
@@ -41,28 +42,38 @@ class IdentityState {
     ///   - configSharedState: the current configuration shared state available at registration time
     ///   - event: The `Event` triggering the bootup
     /// - Returns: True if we should share state after bootup, false otherwise
-    func bootupIfReady(configSharedState: [String: Any], event: Event) -> Bool {
+    func boot(event: Event) -> Bool {
```

Comment:
I think we should keep the control for hasBooted inside of this function and with the change below I think we can remove the bootUp func, 
e.g.
if hasBooted {
return true
}